tools/kicad_rs/src/parser.rs

```diff
@@ -7,17 +7,37 @@ use kicad_parse_gen::schematic as kicad_schematic;
 
 use crate::types::*;
 
+type ParseResult<T> = Result<T, Box<dyn Error>>;
```

Comment:
If we make an `utils` file, this would make a pretty good `DynamicError` type

Reply:
Or `DynamicResult`, for now it can live here, but we should definitely do that move if it's also used elsewhere.

Reply:
`DynamicResult` Oh, yeah that was what I meant :joy: